app/view_models/assignment_repo_view.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class AssignmentRepoView < SharedAssignmentRepoView
+  attr_reader :assignment_repo
+
+  delegate :github_user, to: :assignment_repo
+
+  def avatar_url
+    avatar_url_for(github_user, 96)
+  end
+
+  def user_login
```

Comment:
What do you think about updating this to use `github_login` instead of `user_login`